src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -502,57 +502,152 @@ private string JoinPackageNames(IEnumerable<Package> pkgs)
         /// Warn about duplicate package conflicts depending on whether the package is 
         /// a built-in package and whether versions match.
```

Comment:
Can we change this description too? I think previously this function was meant to detect a conflict with the main package only and now (if I understand correctly), it is detecting conflicts with both the main package as well as its dependencies, correct? If so, it would be nice to make that distinction clear in the comments here.